### PR TITLE
FOGL-2976: New logs tab for notification under logs section

### PR DESCRIPTION
**Pull request by mshariq-nerd**
Screen Shot 2019-07-29 at 5 58 07 PM

Screen Shot 2019-07-29 at 6 33 42 PM

Screen Shot 2019-07-29 at 6 33 58 PM

Response return by http://localhost:8081/foglamp/audit?limit=20&skip=0&source=NTFSN&severity=
`
{
    "audit": [
        {
            "details": {
                "name": "test"
            },
            "severity": "INFORMATION",
            "source": "NTFSN",
            "timestamp": "2019-07-29 11:40:44.952"
        },
        {
            "details": {
                "name": "test"
            },
            "severity": "INFORMATION",
            "source": "NTFSN",
            "timestamp": "2019-07-29 11:39:39.707"
        }
    ],
    "totalCount": 2
}
`

### CODE REVIEW

File: src/app/components/core/notifications/notification-log/notification-log.component.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+bottom: 0px;
+margin: 0px;
+padding: 4px;
+background: white;
+}
+
+.pagination-section {
+padding: 0.5rem;
+}
+
```
Comment (monika-Dianomic) [inline src/app/components/core/notifications/notification-log/notification-log.component.css]: It seems that above css classes are not being used anywhere. Please remove it.

File: src/app/components/core/notifications/notification-log/notification-log.component.ts [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    }
+    if (this.page !== 1) {
+      this.page = 1;
+      this.tempOffset = this.offset;
+    }
+    if (limit === '' || limit === 0 || limit === null || limit === undefined) {
+      limit = this.DEFAULT_LIMIT;
+    }
+    this.limit = limit;
+    console.log('Limit: ', this.limit);
```
Comment (monika-Dianomic) [inline src/app/components/core/notifications/notification-log/notification-log.component.ts]: Please remove log

File: src/app/components/core/notifications/notification-log/notification-log.component.ts [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+    }
+
+    if (this.page !== 1) {
+      this.page = 1;
+    }
+    if (offset === null || offset === undefined) {
+      offset = 0;
+    }
+    this.offset = offset;
+    console.log('Offset: ', this.offset);
```
Comment (monika-Dianomic) [inline src/app/components/core/notifications/notification-log/notification-log.component.ts]: Please remove log

**Comment by billehunt** [comment]
When the user selects the Notification Log tab on the left, does the default view show all Notifications entries? Based on the screenshots above, I imagine it's actually showing only one one type (like NTFAD).

We should be able to see one log with all Notification info.

File: src/styles.css [hunk 4]
```diff
@@ -15,7 +15,7 @@ html,
   background-color: #f5f5f5 !important;
 }
 
-@media only screen and (min-height: 600px) {
+@media only screen and (min-height: 650px) {
```
Comment (praveen-garg) [inline src/styles.css]: should it not be 768? https://bulma.io/documentation/overview/responsiveness/

**Pull request merged**